docs/configuration.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Configuration file'
+description: 'Use a JSON file to configure your tests'
+---
+
+Empirical uses a JSON configuration file, which is located at `empiricalrc.json`, to describe
```

Comment:
Empirical uses a JSON configuration file, which is located at `empiricalrc.json`,
> located sounds like a directory but actually its a file.